input/ig.xml

```diff
@@ -599,31 +599,31 @@
 				<reference value="Practitioner/bobrester-bob"/>
 			</reference>
 			<name value="Practitioner - Dr. Bob Bobrester"/>
-			<description value="Dr. Bob Bobrester"/>
+			<description value="Shows an example of a practitioner for the *AU Core Practitioner* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Practitioner/sallie-sutherland"/>
 			</reference>
 			<name value="Practitioner - Dr. Sallie Sutherland"/>
-			<description value="Dr. Sallie Sutherland"/>
+			<description value="Shows an exaple of a practitioner with HPI-I for the *AU Core Practitioner* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Practitioner/chau-fryer"/>
 			</reference>
 			<name value="Practitioner - Dr. Chau FRYER"/>
-			<description value="Dr. Chau Fryer"/>
+			<description value="Shows an example of practitioner with HPI-I for the *AU Core Practitioner* profile."/>
```

Comment:
add "a" in front of "practitioner" in this and the following example for Darleen.

Reply:
Done